Jenkinsfile

```diff
@@ -6,7 +6,7 @@ node('docker') {
 
     stage('Cleaning') {
         try {
-            sh "docker rm -f chain broker blockchain-adapter-mongo > /dev/null 2>&1 && " +
+            sh "docker rm -f ibaa-chain ibaa-broker ibaa-blockchain-adapter-mongo > /dev/null 2>&1 && " +
```

Comment:
Docker-compose relies on project name, AKA parent directory name while Jenkins builds have different directory names for different features. Do you see?

Reply:
Thought, this part ensures (+) the build will have a fresh setup but (-) it will kill IBAA builds already in progress.
This sounds fine as a first try.